docs/installation/installation-detailed.rst

```diff
@@ -22,8 +22,7 @@ Detailed installation guide
 
 .. admonition:: CAVEAT LECTOR
 
-   Invenio v3.0 alpha is a bleeding-edge developer preview version that is
-   scheduled for public release in Q3/2016.
+   Invenio v3.0 alpha is a bleeding-edge developer preview version.
```

Comment:
If we are amending and/or removing Q3/206, then there are a few more occurrences:

```console
$ git grep 'Q3/2016'
docs/architecture/system-overview.rst:   scheduled for public release in Q3/2016.
docs/installation/installation-detailed.rst:   scheduled for public release in Q3/2016.
docs/introduction/releases.rst:set of standalone independent Python packages. *(ETA: Q3/2016)*
```

I'd rather replace Q3/2016 by Q1/2017, as it is good to leave some kind of public ETA indication... even if we have to postpone the date.